action.yaml

```diff
@@ -87,6 +87,31 @@ runs:
         $BINARY --version
       done
 
+  # Make sure we have mkfs.xfs that can handle spaces in protofiles.
+  # TODO: Drop when we move to the next Ubuntu LTS.
+  - name: Update xfsprogs
+    shell: bash
+    working-directory: ${{ github.action_path }}
+    run: |
+        sudo apt-get install --assume-yes --no-install-recommends \
```

Comment:
I'm very much in favour of indenting four spaces, but if you want to indent only two, this is indented too far.